src/components/Input/index.tsx

```diff
@@ -1,23 +1,18 @@
 import React from 'react';
 import { View, StyleSheet, TextInput } from 'react-native';
+import { InputProps } from './types';
```

Comment:
I think it's better to use more conventional solutions:
1. Either store InputProps in this file
2. Or create index.d.ts and place there

You can read about it here:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html